### DISCUSSION TITLE
[$20] Finish #1401, Security 911 (Backup Requested!)

### DISCUSSION THREAD

**Discussion by ReturnToZender** [Bubberstation/Bubberstation#1886] [ideas/bounties]
#1401, Security 911 (Backup Requested) creates the ability to use the Sec hailer to call for backup. Due to a bug with hearing sec comms through the hailer, it couldn't be finished. Now, I'll pay you to fix that bug so it's ready for merging. If it takes more time than anticipated, bounty amount is adjustable.

**Comment by StrangeWeirdKitten** [comment]
My hurdle with the original PR was that there was a TG bug with radios overall being hearable even when you turn off broadcasting.

**Comment by Odairu** [comment]
This is completed, waiting on TG to accept the bug fix I PRed there, then it can be merged when that makes its way here 👍

**Comment by ReturnToZender** [comment]
Work done, closing
